site/content/docs/4.3/content/tables.md

```diff
@@ -84,12 +119,10 @@ You can also invert the colors—with light text on dark backgrounds—with `.ta
 </table>
 {{< /example >}}
 
-### Striped rows
-
-Use `.table-striped` to add zebra-striping to any table row within the `<tbody>`.
+And because this happens with utilities, you have some flexibility in how you style your tables relative to their context. For example, if your page is already dark gray with lighter text, you don't need to set every utility on the `<table>`—just `.border-gray-700`.
```

Comment:
It took a while before I noticed what was going on here, I was just screening the page without reading the intro text and thought padding was somehow added to the table. I would emphasise the dark page notice more (bold text?) or add what utilities were added to the parent (because you can't see them in the snippet)